modules/admin_manual/pages/maintenance/migrating_to_kiteworks.adoc

```diff
@@ -336,22 +425,22 @@ By default, rclone transfers 4 files in parallel. This creates little load on th
 
 Performance can be tuned with the environment variable `RCLONE_TRANSFERS` which defines the number of concurrent file uploads.
 
-The following example command is using 32 parallel transfers:
+The following example command is using 16 parallel transfers:
```

Comment:
drop this please from the official docs - we only tested with default settings.

Reply:
fixed